projects/packages/connection/legacy/class-jetpack-options.php

```diff
@@ -203,7 +227,7 @@ public static function get_option( $name, $default = false ) {
 		 * @param string $name Option name, _without_ `jetpack_%` prefix.
 		 * @return string $value, unless the filters modify it.
 		 */
-		return apply_filters( 'jetpack_options', self::get_option_from_database( $name, $default ), $name );
+		return apply_filters( 'jetpack_options', $database_value, $name );
```

Comment:
Wouldn't it be better to leave the original code instead of storing the result in a variable, just to be used here? Or will it be used (`$database_value` elsewhere in the code?